### PR TITLE
Update scancode

### PR DESCRIPTION
**Pull request by nellshamrell**
This corresponds with the [crawler pull request which update scancode](https://github.com/clearlydefined/crawler/pull/459). These two pull requests should be deployed together.

### CODE REVIEW

File: providers/summary/scancode.js [hunk 1]
```diff
@@ -47,9 +47,17 @@ class ScanCodeSummarizer {
...
+    switch (scancodeVersion) {
+      case '2.2.1':
+      case '2.9.2':
+      case '2.9.8':
+      case '3.0.0':
+      case '3.0.2':
+        return SPDX.normalize(get(harvested, 'content.summary.packages[0].declared_license'))
+      case '30.1.0':
+        return SPDX.normalize(get(harvested, 'content.summary.packages[0].declared_license[0]'))
+    }
```
Comment (jeffwilcox) [inline providers/summary/scancode.js:60]: should there be a default/fallthrough clause to throw or err out if a different version just happens to make it to this codepath?

File: test/providers/summary/scancode.js [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 const assert = require('assert')
 const summarizer = require('../../../providers/summary/scancode')()
-summarizer.logger = { info: () => {} }
+summarizer.logger = { info: () => { } }
 const fs = require('fs')
 const path = require('path')
 const { get, uniq, flatten } = require('lodash')
 
-const scancodeVersions = ['2.2.1', '2.9.2', '2.9.8', '3.0.0', '3.0.2']
+const scancodeVersions = ['2.2.1', '2.9.2', '2.9.8', '3.0.0', '3.0.2', '30.1.0']
```
Comment (jeffwilcox) [inline test/providers/summary/scancode.js:11]: another nit, feel free to leave be; depending on your coding guidelines, I know I'd prefer to have all the array items here be their own unique lines, to make future diffs more clear what is changing

**Review by jeffwilcox — APPROVED** [review]
did add a minor comment

File: providers/summary/scancode.js (hunk 1 above)
Comment (nellshamrell) [reply providers/summary/scancode.js:60]: that's a good note - taking a look

File: test/providers/summary/scancode.js (hunk 2 above)
Comment (nellshamrell) [reply test/providers/summary/scancode.js:11]: I think that may make our linter mad, but I will give it a try

File: test/fixtures/scancode/0.0.0/npm-basic.json [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      "self": {
+        "href": "urn:npm:npmjs:-:glob:revision:7.1.2:tool:scancode:2.2.1",
+        "type": "resource"
+      },
+      "siblings": {
+        "href": "urn:npm:npmjs:-:glob:revision:7.1.2:tool:scancode",
+        "type": "collection"
+      }
+    },
+    "version": "2.2.1",
```
[the remark this answers is not in the document]
Comment (nellshamrell) [reply test/fixtures/scancode/0.0.0/npm-basic.json:16]: I just wanted a placeholder :)

**Pull request merged**